cluster/extensions-javadsl/src/main/java/com/lightbend/lagom/javadsl/cluster/projections/DesiredState.java

```diff
@@ -17,6 +17,7 @@
 @ApiMayChange
 public final class DesiredState {
 
+  // TODO: use PSequence instead
```

Comment:
I believe that this code should one day move to akka-projection and it will be better to not add a dependency on PCollection. Don't get me wrong, I like PCollections, but we should minimize external dependencies. 

Since we are in control of this class we can make sure that nobody has access to this mutable list.

Reply:
I went for the impl closest to Akka's rules for JAva modeling and then realized that broke the Lagom rules (use PCollections).

I think we can drop the `TODO`